### PR TITLE
updated mycode in feature branch

### PR DESCRIPTION
**Pull request by mohansgithub**
Please check and review

### CODE REVIEW

File: src/main/webapp/WEB-INF/views/home.jsp [hunk 1]
```diff
@@ -11,10 +11,11 @@
...
 			<h1>DevOps Traning Mohanraj Vellingiri</h1>
-			<h1> Devops Training Nallaperumal TN	</h1>
-			<h1> Devops Training Nallaperumal TN	</h1>
-			<h1>2 DevOps Training Mohit Kambam</h1>
-			</h1> DevOps Training 12 09 2020 Mohit Kambam</h1>
+			<h1>Devops Training Nallaperumal TN	</h1>
+			<h1>Devops Training Nallaperumal TN	</h1>
+			<h1>DevOps Training Mohit Kambam</h1>
+			<h1> DevOps Training 12 09 2020 Mohit Kambam</h1>
+			My New Code is here
```
Comment (mydevopstrainings1) [inline src/main/webapp/WEB-INF/views/home.jsp:18]: Please update the code with Training name

Comment (mydevopstrainings1) [reply src/main/webapp/WEB-INF/views/home.jsp:18]: Use capital letters

**Review by mydevopstrainings1 — CHANGES_REQUESTED** [review]
Please updated the code and  send me back

File: src/main/webapp/WEB-INF/views/home.jsp (hunk 1 above)
Comment (mohansgithub) [reply src/main/webapp/WEB-INF/views/home.jsp:18]: Ok

Comment (mohansgithub) [inline src/main/webapp/WEB-INF/views/home.jsp:18]: Thank you updated

**Comment by mydevopstrainings1** [comment]
Merged

**Pull request merged**